### DISCUSSION TITLE
Enhancement:a button of default controls

### DISCUSSION THREAD

**Discussion by dowks789** [FunkinCrew/Funkin#1644] [Ideas]
Please add this each game has it and apart it would be very useful for example with me who wanted to play with my xbox control week 7 since I passed the others before but out of curiosity I wanted to see the new options button (game in downloaded version) and I got into controller controls and put button A on the left control but I couldn't remove it. The left of  t pad itched but it only showed a question mark. I used the lever and it didn't detect the left. Please add the button to restore controls (I use the translator to translate it, I speak Spanish)

**Comment by ghost** [comment]
No hablo español, pero si vas a utilizar un traductor, te recomiendo DeepL. He descubierto que es el que mejor funciona. (in English: I don't speak spanish, but if you are going to use a tranlator, I recommend DeepL. I found that it works best.)

**Comment by Windows1Door2Daniel** [comment]
> No hablo español, pero si vas a utilizar un traductor, te recomiendo DeepL. He descubierto que es el que mejor funciona. (in English: I don't speak spanish, but if you are going to use a tranlator, I recommend DeepL. I found that it works best.)

the english is not a problem

**Comment by GOOBOZA** [comment]
S, TF, U
to everyone who think its bad idea
